CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## main / unreleased
 
 ### Grafana Mimir
-
+* [BUGFIX] Helm-Chart: fix route to service port mapping. #4727 
```

Comment:
Please move to https://github.com/grafana/mimir/blob/main/operations/helm/charts/mimir-distributed/CHANGELOG.md  and note the ordering of items, bugfix entry goes to the end.